File: src/main/java/cn/dsnbo/bedrockplayersupport/command/TpFormCommand.java
package cn.dsnbo.bedrockplayersupport.command;

import cn.dsnbo.bedrockplayersupport.BedrockPlayerSupport;
import cn.dsnbo.bedrockplayersupport.form.MainForm;
import org.bukkit.Bukkit;
import org.bukkit.command.Command;
import org.bukkit.command.CommandExecutor;
import org.bukkit.command.CommandSender;
import org.bukkit.entity.Player;
import org.geysermc.floodgate.api.FloodgateApi;

/**
 * @author DongShaoNB
 */
public class TpFormCommand implements CommandExecutor {

  @Override
  public boolean onCommand(CommandSender sender, Command command, String label, String[] args) {
    if (sender instanceof Player player) {
      if (FloodgateApi.getInstance().isFloodgatePlayer(player.getUniqueId())) {
        if (Bukkit.getOnlinePlayers().size() > 1) {
          MainForm.getInstance().openBedrockTeleportForm(player);
        } else {
          player.sendMessage(BedrockPlayerSupport.getLanguageConfigManager().getConfigData().noOtherOnlinePlayer());
        }
      } else {
        player.sendMessage(BedrockPlayerSupport.getLanguageConfigManager().getConfigData().notBedrockPlayer());
      }
    }
    return false;
  }

}

File: src/main/kotlin/cc/dsnb/bedrockplayersupport/command/TpFormCommand.kt
package cc.dsnb.bedrockplayersupport.command

import cc.dsnb.bedrockplayersupport.BedrockPlayerSupport
import org.bukkit.Bukkit
import org.bukkit.command.Command
import org.bukkit.command.CommandExecutor
import org.bukkit.command.CommandSender
import org.bukkit.entity.Player

/**
 * @author DongShaoNB
 */
class TpFormCommand : CommandExecutor {

    override fun onCommand(sender: CommandSender, command: Command, label: String, args: Array<String>): Boolean {
        if (sender is Player) {
            if (BedrockPlayerSupport.floodgateApi.isFloodgatePlayer(sender.uniqueId)) {
                if (Bukkit.getOnlinePlayers().size > 1) {
                    BedrockPlayerSupport.mainForm.openBedrockTeleportForm(sender)
                    return true
                } else {
                    sender.sendMessage(
                        BedrockPlayerSupport.miniMessage
                            .deserialize(
                                BedrockPlayerSupport.langConfigManager.getConfigData()
                                    .noOtherOnlinePlayer()
                            )
                    )
                }
            } else {
                sender.sendMessage(
                    BedrockPlayerSupport.miniMessage.deserialize(
                        BedrockPlayerSupport.langConfigManager.getConfigData().notBedrockPlayer()
                    )
                )
            }
        }
        return false
    }

}